docs/installation.md

```diff
@@ -40,6 +48,26 @@ Note that by default, Jupyter Notebook will not be able to open a browser window
 WSL2 [does support](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gpu-compute) GPU compute, at least with nvidia cards, but setting it up [takes some work](https://developer.nvidia.com/cuda/wsl).
 :::
 
+
+(installation/m1)=
+
+#### macOS with M1 chips
+
+Apple Silicon (M1 mac, `osx-arm64`) is supported, but only through Rosetta because (as of August 2022) some upstream dependencies are not yet built on it.
+To configure `conda` to [use Rosetta](https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon) when creating an environment (or modifying one, such as installing new packages into it), prepend the command with `CONDA_SUBDIR=osx-64`:
+
+```shell-session
+$ CONDA_SUBDIR=osx-64 conda install -c conda-forge openff-toolkit
+```
+
+To make this setting the global default, update the conda config via
+
+```
+$ conda config --env --set subdir osx-64
```

Comment:
```suggestion
$ conda config --system --set subdir osx-64
```

I think this is the correct command for system-wide configuration?

Reply:
I honestly have no idea. I had a hard time finding information on this using official conda docs and learned everything I know about this trick from comments by people not employed by Anaconda (Jaime, Isuru). Maybe I'm using the wrong search terms ...


My intent here was to be helpful to the "just get stuff working" crowd, assuming that we're not the only project they'll run into this headache with. I'm not sure it's worth expanding the scope from "how to get The OpenFF Toolkit working on your new computer" to "cool tips and tricks to help your general Conda experience"

Reply:
This is getting close to the wire but I'm comfortable merging this line. I tested it locally by deleting my `$CONDA_PREFIX_1/.condarc` file, running `$ conda config --system --set subdir osx-64`, checking the RC file again to see that `subdir` was set, and also running a `$ mamba update --all -c conda-forge` and seeing no ARM builds pulled down.